apps/api/src/server/app/repositories/appeal.repository.test.js

```diff
@@ -0,0 +1,18 @@
+import test from 'ava';
```

Comment:
Why are we using and what is "ava" ? I thought the plan was to use jest for testing?

Reply:
I suggested to Mayya as an alternative as Jest is not really maintained anymore and it also doesn't support ESM. Jest is mostly used for Web stuff and not necessarily for Node.

[Ava](https://github.com/avajs/ava) is an alternative that is designed for Node, runs tests concurrently and is as easy to use as any other testing library I guess.

If the preferred lib is Jest we can convert back to it as the refactor is minimal.